### ISSUE TITLE
PHP version 8.0.0 not supported

### ISSUE THREAD

**Issue by quadratz** [noplanman/php-telegram-bot-manager#60]
#### Required Information

|  ?                               |  !
| ---                              | ---
| Operating system                 | Android 10 (Using termux)
| PHP Telegram Bot Manager version | 1.5.0
| PHP Telegram Bot version         | 0.64.0
| PHP version                      | 8.0.0
| MySQL version                    | none
| Update Method                    | none
| Self-signed certificate          | none
| RAW update (if available)        | `{...}`

#### Summary
Could not update the Telegram Bot Manager because it was expecting php version 7.1 instead of 8.0.0 (the newer version)

### Current behaviour
This error occured when `$composer update` is executed.
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - php-telegram-bot/telegram-bot-manager[1.4.0, ..., 1.5.0] require php ^7.1 -> your php version (8.0.0) does not satisfy that requirement.
    - Root composer.json requires php-telegram-bot/telegram-bot-manager ^1.4 -> satisfiable by php-telegram-bot/telegram-bot-manager[1.4.0, 1.5.0].
```

#### How to reproduce
1. Run `$composer update`
2. Error occured.

#### Expected behaviour

PHP version 8.0.0 was expected to satisfy the requirement.

**Comment by noplanman** [comment]
I'm busy preparing the next version release of the core library, which will support PHP 8.
Then I'll bump this repo too 👍

**Comment by noplanman** [comment]
@CikiMomogi Version 1.6.0 has been released, now supporting the core version 0.70.* and PHP 8 👍
